CRM/Admin/Form/Setting/UF.php

```diff
@@ -79,7 +79,7 @@ public function buildQuickForm() {
       $tablePrefixes .= "\n  'default' => '$drupal_prefix',";
       $prefix = "";
       if ($config->dsn != $config->userFrameworkDSN) {
-        $prefix = "`{$dsnArray['database']}`.";
+        $prefix = "{$dsnArray['database']}.";
         if ($config->userFramework === 'Backdrop') {
```

Comment:
@pradpnayak I'm thinking that these 3 lines probably can also go in that case thoughts @MikeyMJCO @herbdool @MegaphoneJon ?

Reply:
You'd still need them for drupal 7/8 right? (e.g. if you have no control over the database name and it's a reserved word or something). What's the actual cause? Does drupal 9 automatically add backticks if you use the squiggly brackets?

Reply:
Do we need to switch the format based on the UF? Have we finally hit the first place where we need `civicrm-drupal-9` :-D

Reply:
Also https://www.drupal.org/project/drupal/issues/3106531 (so civicrm-drupal-10 ?)

Reply:
Unclear after reading that issue - does that essentially mean that Views integration won't be possible with separate databases in D10?

Reply:
Maybe - also noticed this: https://github.com/eileenmcnaughton/civicrm_entity/pull/306#issuecomment-842814747

Reply:
Note: https://github.com/eileenmcnaughton/civicrm_entity/issues/313